resources/views/backend/voucher/index.blade.php

```diff
@@ -76,6 +76,8 @@ class="d-none d-sm-inline-block btn btn-sm btn-primary shadow-sm">
                     {data: 'expires_at', name: 'expires_at'},
                     {data: 'created_at', name: 'created_at', searchable: false},
                     { data: function ( row ) {
+                            if(row.redeemed_at !== null)
+                                return ``;
```

Comment:
If you don't want tat the redeemed would be shown [VoucherController](https://github.com/Devsome/silkroad-laravel/blob/master/app/Http/Controllers/Backend/VoucherController.php#L29) for more performance.  I will do a "toggle" redeemed Vouchers. Maybe an Admin need to search or see how many Vouchers are already used.

Reply:
The change is about hiding "deleting action" from redeemed voucheers, it throws an exception if you do.

I didn't check the tables but had something to do with an associated/ForeignKey I think.

Reply:
Then my bad. This is all good 👍